### DISCUSSION TITLE
Members

### DISCUSSION THREAD

**Discussion by jdhaines** [aiaa-deic/devsecops#9] [General]
I think it might be useful to get a list of the subcommittee members and basic information.  I'd like to have a page on the website that includes this in a nice way in order to recognize anyone.  If you would like to be a part of that page, post your information in a reply below.  Some good basic information might be:

Name, Company, Title, Hobbies, and a picture.  I'll go first below!

**Comment by jdhaines** [comment]
|Field|Value|
|---|---|
|Name|Josh Haines|
|Company|Rolls-Royce|
|Title|DevSecOps Capability Lead & Software Factory Founder|
|Hobbies|Coding Side Projects, Cycling, Chess|
|Picture|<img src="https://avatars.githubusercontent.com/u/3834260?v=4" width=200 />|

**Comment by ba-napitts** [comment]
- Name
  - Nathan Pitts
- Company
  - Boeing
- Title
  - Senior Systems Engineer
- Hobbies
  - Too many ... Piano, Coffee, Skiing, Swimming, Books and Rain, Games
- Picture
  - todo
